ledger/sandbox-classic/BUILD.bazel

```diff
@@ -11,242 +11,120 @@ load("//ledger/ledger-api-test-tool:conformance.bzl", "server_conformance_test")
+    "//libs-scala/ports",
+    "//libs-scala/resources",
+    "@maven//:ch_qos_logback_logback_classic",
+]
+
 alias(
     name = "sandbox-classic",
     actual = "sandbox-classic-ce",
     visibility = ["//visibility:public"],
 )
 
-[
-    da_scala_library(
-        name = "sandbox-classic-{}".format(edition),
-        srcs = glob(["src/main/scala/**/*.scala"]),
-        # Do not include logback.xml into the library: let the user
-        # of the sandbox-as-a-library decide how to log.
-        resources = ["//ledger/sandbox-common:src/main/resources/banner.txt"],
-        scala_deps = [
-            "@maven//:com_github_scopt_scopt",
-            "@maven//:com_typesafe_akka_akka_actor",
-            "@maven//:com_typesafe_akka_akka_stream",
-            "@maven//:org_scalaz_scalaz_core",
-        ],
-        tags = ["maven_coordinates=com.daml:sandbox-classic:__VERSION__"],
-        visibility = [
-            "//visibility:public",
-        ],
-        runtime_deps = [
-            "@maven//:ch_qos_logback_logback_classic",
-            "@maven//:ch_qos_logback_logback_core",
-            "@maven//:com_h2database_h2",
-            "@maven//:org_postgresql_postgresql",
-        ],
-        deps = [
-            "//daml-lf/archive:daml_lf_1.dev_archive_proto_java",
-            "//daml-lf/archive:daml_lf_archive_reader",
-            "//daml-lf/data",
-            "//daml-lf/language",
-            "//language-support/scala/bindings",
-            "//ledger/caching",
-            "//ledger/error",
-            "//ledger/ledger-api-auth",
-            "//ledger/ledger-api-common",
-            "//ledger/ledger-api-domain",
-            "//ledger/ledger-api-health",
-            "//ledger/ledger-configuration",
-            "//ledger/ledger-resources",
-            "//ledger/metrics",
-            "//ledger/participant-integration-api",
-            "//ledger/participant-state",
-            "//ledger/participant-state/kvutils/app",
-            "//ledger/sandbox-common:sandbox-common-{}".format(edition),
-            "//ledger-service/cli-opts",
-            "//ledger/sandbox-on-x",
-            "//libs-scala/build-info",
-            "//libs-scala/contextualized-logging",
-            "//libs-scala/logging-entries",
-            "//libs-scala/ports",
-            "//libs-scala/resources",
-            "//libs-scala/resources-akka",
-            "//libs-scala/resources-grpc",
-            "@maven//:ch_qos_logback_logback_classic",
-            "@maven//:com_typesafe_config",
-            "@maven//:io_dropwizard_metrics_metrics_core",
-            "@maven//:org_slf4j_slf4j_api",
-        ],
-    )
-    for edition in [
-        "ce",
-        "ee",
-    ]
-]
+# TODO sandbox-classic cleanup: De-alias
+alias(
+    name = "sandbox-classic-scala-tests-lib",
+    actual = "//ledger/sandbox-on-x:sandbox-on-x-test-lib",
+    visibility = ["//visibility:public"],
+)
+
+da_scala_library(
+    name = "sandbox-classic-ce",
```

Comment:
At this point the `-ce` suffix can probably be removed and the alias above removed.

Reply:
done 👍